.github/workflows/manual-web-client.yml

```diff
@@ -53,3 +54,28 @@ jobs:
 
       - name: Deploy
         run: yarn deploy:web-client:${{ github.event.inputs.environment }}
+      - name: Get current lerna version
+        id: lerna-version
+        uses: saulonunesdev/lerna-get-version-action@v1.0.4
+      - name: Create decompressed copies of assets for Sentry
+        shell: bash
+        run: |
+          cd packages/web-client/tmp
+          mkdir -p deploy-dist-uncompressed/assets
+          cd deploy-dist/assets
+          echo *.js *.map | tr ' ' '\n' | xargs -n1 -I{} cp "{}" "../../deploy-dist-uncompressed/assets/{}.br"
+          cd ../../deploy-dist-uncompressed/assets
+          brotli --rm --decompress *.br
+      - name: Create Sentry release
+        uses: getsentry/action-release@v1
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.WEB_CLIENT_SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: cardstack
+          SENTRY_PROJECT: web-client
+        with:
+          environment: ${{ github.event.inputs.environment }}
+          ignore_empty: true
+          ignore_missing: true
+          sourcemaps: packages/web-client/tmp/deploy-dist-uncompressed
+          version: web-client-${{ github.sha }}@${{ steps.lerna-version.outputs.lerna-version }}
+          url_prefix: https://${{ fromJSON('["app-staging.stack.cards", "app.cardstack.com"]')[github.event.inputs.environment != 'staging'] }}
```

Comment:
This strange formation is a [workaround](https://github.community/t/do-expressions-support-ternary-operators-to-change-their-returned-value/18114/4) for the lack of a ternary operator. But is there a better way to determine the domain for S3 assets based on the environment? I couldn’t find anywhere to extract these from. The `url_prefix` is required for Sentry to properly look up the artifacts uploaded as part of the release when resolving a stacktrace.